tox.ini

```diff
@@ -75,7 +75,7 @@ deps =
 commands =
     # Download patroni resource to use in the charm deployment.
     sh -c 'stat patroni.tar.gz > /dev/null 2>&1 || curl "https://github.com/zalando/patroni/archive/refs/tags/v2.1.3.tar.gz" -L -s > patroni.tar.gz'
-    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
+    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} --asyncio-mode=auto
```

Comment:
Maybe add to pyproject.toml instead?

```toml
[tool.pytest.ini_options]
minversion = "6.0"
log_cli_level = "INFO"
asyncio_mode = "auto"
```

Reply:
Changed in https://github.com/canonical/postgresql-operator/pull/10/commits/4bdb47ed4156478d5f67dfa98e8b67c60a5fabaf.